torch/onnx/symbolic.py

```diff
@@ -1097,11 +1097,11 @@ def retrieve_state(x, start, end):
             # The ONNX RNN/GRU/LSTM produce an output of dimensions
             #   seq_len, num_directions, batch, hidden_size
             # We have to convert to match pytorch's expected
-            #   seq_len, batch, hidden_size * num_directions
+            #   seq_len, batch, num_directions * hidden_size
             # by first moving num_directions to the end with
```

Comment:
also swap hidden_size/num_directions in the remainder of this comment?

Reply:
Done